vita3k/kernel/src/load_self.cpp

```diff
@@ -70,13 +70,19 @@ static bool load_var_imports(const uint32_t *nids, const Ptr<uint32_t> *entries,
 
         VarImportsHeader *const var_reloc_header = reinterpret_cast<VarImportsHeader *>(entry.get(mem));
         const auto var_reloc_entries = static_cast<void *>(var_reloc_header + 1);
+        const uint32_t reloc_size = (var_reloc_header->reloc_data_size > sizeof(VarImportsHeader)) ? (var_reloc_header->reloc_data_size - sizeof(VarImportsHeader)) : 0;
 
         Address export_address;
         const ExportNids::iterator export_address_it = kernel.export_nids.find(nid);
+        const char *const name = import_name(nid);
         if (export_address_it != kernel.export_nids.end()) {
             export_address = export_address_it->second;
+            if (kernel.late_binding_infos.contains(nid)) {
```

Comment:
The call to contains is not thread safe, putting the lock_exclusive above is the easiest way to fix this.